deps/v8/tools/gen-postmortem-metadata.py

```diff
@@ -139,14 +167,31 @@
         'value': 'ScopeInfo::kStackLocalCount' },
     { 'name': 'scopeinfo_idx_ncontextlocals',
         'value': 'ScopeInfo::kContextLocalCount' },
+    { 'name': 'scopeinfo_idx_ncontextglobals',
+        'value': 'ScopeInfo::kContextLocalCount' },
```

Comment:
Oh sh%t. I'll try to fix it during next iteration. It is not used anywhere just yet.